versions/1/integration/data_stream/manifest.spec.yml

```diff
@@ -143,6 +143,11 @@ spec:
     hidden:
       description: Specifies if a data stream is hidden
       type: boolean
+    allow_routing:
+      description: When set to true, the integration is granted data stream privileges for all datasets and namespaces of its type
+      type: boolean
+      items:
+        type: string
```

Comment:
Something is wrong here. The field is boolean, but you defined string items for it as it's an array.